examples/combined_ODNP.py

```diff
@@ -62,8 +64,12 @@
 config_dict['FIR_rep'] = FIR_rep
 # }}}
 # {{{Power settings
-dB_settings = gen_powerlist(
-    config_dict["max_power"], config_dict["power_steps"] + 1, three_down=True
+dB_settings = Ep_spacing_from_phalf(
+    est_phalf=config_dict["guessed_phalf"] / 4,
+    max_power=config_dict["max_power"],
+    p_steps=config_dict["power_steps"] + 1,
+    min_dBm_step=config_dict["min_dBm_step"],
+    three_down=True,
```

Comment:
now generating power axis with new function to get more data points in linear regime